.changeset/real-jobs-own.md

```diff
@@ -0,0 +1,6 @@
+---
+"miniflare": patch
+"wrangler": patch
+---
+
+feat: Implement Proxy Worker behaviour we need for Workers+Assets
```

Comment:
This is describing the implementation rather than the effect it has on the user's experience.

Reply:
true! will reword